tests/settings/e2e/features/wallet-settings.feature

```diff
@@ -78,22 +78,11 @@ Feature: Wallet Settings
     And I click outside "name" input field
     Then I should see new wallet name "キュビズム"
 
-  @shelley
-  Scenario: User force Rewards Wallet resync
+  @skip
```

Comment:
@tomothespian this test should work on ITN builds. Please check and update this tag to run the test only on ITN builds if that is the case 🙏